src/gui/qgsmaptool.h

```diff
@@ -109,9 +109,6 @@ class GUI_EXPORT QgsMapTool : public QObject
      */
     enum Flag
     {
```

Comment:
We will need to leave the transient value here, but change the docs to say that it's deprecated and no longer has any effect.